TriH/THEMMOI_BN_133.js

```diff
@@ -0,0 +1,60 @@
+describe('THEMMOI_BN', function() {
+
+	beforeEach(function(){ //Đăng nhập và chuyển tới trang thêm bệnh nhân
+
+		cy
+
+			.visit("http://52.187.8.102/signin")
```

Comment:
các em không hard code cái địa chỉ, dùng cy.visit(Cypress.env("host")) // points to a dynamic env var
xem ở đây: https://docs.cypress.io/guides/guides/environment-variables.html#

Reply:
Em sẽ cố gắng thử ạ, Em cám ơn thầy